pkg/server/providers.go

```diff
@@ -13,6 +13,8 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+var AllProviderRegistered = false
```

Comment:
Instead of this, I suggest that the provider manager has an `IsInitialized` function that will return true if it is initialized.

Then you can expose an endpoint in the provider controller that will return the "health" of the providers